### PR TITLE
More messaging refactor

### PR DESCRIPTION
**Pull request by apple502j**
- More detailed error message to help debug issues
- Remove some raw HTML usage
- Add favicon to popups

### CODE REVIEW

File: webpages/popup/index.html [hunk 1]
```diff
@@ -8,6 +8,7 @@
       @import url("../styles/components/tooltips.css");
       @import url("../styles/colors.css");
     </style>
+    <link rel="icon" href="../../images/icon.png" />
```
Comment (WorldLanguages) [inline webpages/popup/index.html:11]: Users aren't expected to open this file as a tab, so this looks unnecessary.

File: webpages/popup-loader.js [hunk 2]
```diff
@@ -180,3 +180,10 @@ if (window.parent === window) {
   document.body.classList.add("fullscreen");
   document.documentElement.classList.add("fullscreen");
 }
+
+document.head.appendChild(
+  Object.assign(document.createElement("link"), {
+    rel: "icon",
+    href: "../../images/icon.png",
```
Comment (WorldLanguages) [inline webpages/popup-loader.js:187]: Can be shortened to `/images/icon.png`

File: popups/scratch-messaging/popup.js [hunk 3]
```diff
@@ -550,7 +550,7 @@ export default async ({ addon, msg, safeMsg }) => {
             elementObject.loadedComments = true;
           })
           .catch((e) => {
-            if (e instanceof API.HTTPError) {
+            if (e instanceof API.HTTPError && e.code > 400) {
```
Comment (WorldLanguages) [inline popups/scratch-messaging/popup.js:553]: `>=` ?

Comment (apple502j) [reply popups/scratch-messaging/popup.js:553]: Nope, `>`.

File: webpages/popup/index.html (hunk 1 above)
Comment (apple502j) [reply webpages/popup/index.html:11]: Doesn't really matter

**Pull request merged**